mne/gui/_core.py

```diff
@@ -165,12 +165,15 @@ def _load_image_data(self, base_image=None):
                  'head plot, use :ref:`mne.bem.make_scalp_surfaces` '
                  'to add the scalp surface instead of skull from the CT')
             self._head = None
-        if op.exists(op.join(self._subject_dir, 'surf', 'lh.pial')):
-            self._lh = _read_mri_surface(
-                op.join(self._subject_dir, 'surf', 'lh.pial'))
+        for img in ('', '.T1', '.T2', ''):
```

Comment:
you've used this trick elsewhere too, right? The extra `''` at the end just makes the error message cleaner? If that's right, can you just add a comment explaining, commit with `ci skip`, then merge?

Reply:
Yes, that's correct, will do